Makefile.am

```diff
@@ -134,6 +134,15 @@ check_PROGRAMS += \
 # add test scripts
 check_SCRIPTS += $(integration_scripts)
 
+#Cannot be part of $(integration_scripts) since its not EXTRA_DIST
+check_SCRIPTS += test/integration/pkcs11-tool-init-fapi.sh.nosetup
+
+test/integration/pkcs11-tool-init-fapi.sh.nosetup: test/integration/pkcs11-tool-init.sh.nosetup
+	$(AM_V_GEN)(echo "#!/usr/bin/env bash"; echo "export FAPI_PREVIEW=true"; cat $^)>$@ \
+		&& chmod +x "$@"
+
+CLEANFILES += test/integration/pkcs11-tool-init-fapi.sh.nosetup
```

Comment:
Why are we doing all this? Why wouldn't this just be like any other integration test?

Reply:
Well, I wanted to ensure 1-to-1 compatibility between FAPI and non-FAPI and as such I wanted to run the exact same test with FAPI and with non-FAPI.
I guess I want to try the TEST_ENVIRONMENT instead. But that would still need a symlink from .fapi to .nosetup. I guess that's nicer...

Reply:
Ok, so there is no possibility of `FAPI_TEST_ENVIRONMENT`, only a `FAPI_LOG_COMPILER` that I could use to make this work...